site/src/components/breadcrumbs.md

```diff
@@ -0,0 +1,116 @@
+---
+title: Breadcrumbs
+description: The Breadcrumb component in the Atlas Design System
+template: standard
+figmaEmbed: https://www.figma.com/embed?embed_host=share&url=https%3A%2F%2Fwww.figma.com%2Ffile%2FNWYugObOGcAOlekKyjkEkT%2F%25E2%25AD%2590%25EF%25B8%258F-Atlas---Page-Templating%3Fnode-id%3D204%253A1220
+---
+
+# Breadcrumbs
+
+Use breadcrumbs to show navigation from the current location to less specific locations. Note that the final item representing the current page is not presented.
+
+```html
+<nav>
+	<ol class="breadcrumbs">
+		<li class="breadcrumb-item">
+			<a href="#">Domain</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Kingdom</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Phylum</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Class</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Order</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Family</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Genus</a>
+		</li>
+		<li class="breadcrumb-item">
+			<a href="#">Species</a>
+		</li>
+	</ol>
+</nav>
+```
+
+## Removing slashes
```

Comment:
Let's remove this section since it duplicates "Representing the current location" which is the use-case for slashless.

Reply:
Updated!